robot_sort/robot_sort.py

```diff
@@ -96,17 +96,47 @@ def sort(self):
         """
         Sort the robot's list.
         """
-        # Fill this out
-        pass
+        while self.can_move_right(): # we have things on the right
```

Comment:
Great Job Winston, clean and simple, as python guidelines suggest. I sure overcomplicated this when I did it.